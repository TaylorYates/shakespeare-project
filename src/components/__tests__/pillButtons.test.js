import React from 'react';
import {shallow} from 'enzyme';

import PillButtons from '../pillButtons'

describe('<PillButtons/>', () => {
  it('has 6 tabs', () => {
    const wrapper = shallow(<PillButtons />);
    expect(wrapper).toMatchSnapshot();
  })
})
