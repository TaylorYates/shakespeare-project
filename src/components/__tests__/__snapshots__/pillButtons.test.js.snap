// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PillButtons/> has 6 tabs 1`] = `
<div
  className="pillButtons"
>
  <button
    onClick={[Function]}
  >
    All
  </button>
  <button
    onClick={[Function]}
  >
    Act 1
  </button>
  <button
    onClick={[Function]}
  >
    Act 2
  </button>
  <button
    onClick={[Function]}
  >
    Act 3
  </button>
  <button
    onClick={[Function]}
  >
    Act 4
  </button>
  <button
    onClick={[Function]}
  >
    Act 5
  </button>
</div>
`;
